commons/all/src/main/java/org/infinispan/query/remote/client/ProtobufMetadataManagerConstants.java

```diff
@@ -7,7 +7,10 @@
  *
  * @author anistor@redhat.com
  * @since 7.1
+ * @deprecated since 16.0
+ * Use {@link org.infinispan.commons.admin.SchemasAdministration API to interact with the schemas API}
```

Comment:
`org.infinispan.commons.admin.SchemasAdministration` does not exist.

Reply:
removing